app/controllers/concerns/wizard_steps.rb

```diff
@@ -0,0 +1,66 @@
+module WizardSteps
+  extend ActiveSupport::Concern
+
+  included do
+    class_attribute :wizard_class
+    before_action :load_wizard, :load_current_step, except: %i[index completed]
+  end
+
+  def index
+    redirect_to step_path(wizard_class.first_key)
```

Comment:
Thats the idea, ATM it errors on pundit

Reply:
Yeah, honestly I don't think we need this. Maybe if this becomes a requirement in the future, but certainly I'd say it's nice to have for "pro users" rather than something a regular user would come across?